### ISSUE TITLE
Make it more configurable

### ISSUE THREAD

**Issue by sindresorhus** [sindresorhus/gulp-imagemin#63]
Right now we support plugins, but there's no way to disable the built-in plugins.

Would be nice if everything was a plugin.

```js
var gulp = require('gulp');
var pngcrush = require('imagemin-pngcrush');

gulp.task('default', function () {
    return gulp.src('src/images/*')
        .pipe(pngcrush())
        .pipe(gulp.dest('dist'));
});
```

`gulp-imagemin` would still exist as a handy bundle for the common configuration, but it would be possible to use any plugin separately.

@kevva Would it be possible to add support to the imagemin plugins to be used directly in the stream? That would be super nice and much more aligned with what gulp users are used to.

**Comment by iliakan** [comment]
Could be nice if one could disable svgo. It breaks my svgs :)

**Comment by mil** [comment]
@iliakan svgo breaks my svgs too. Quick fix is to add .extensions to opts:
https://github.com/mil/gulp-imagemin/commit/1ecb7b7170f26eb06dc96a55fb86e77b581d9aaf

**Comment by iliakan** [comment]
@mil actually there is a couple of things that breaks.

**Comment by sindresorhus** [comment]
Fixed in 3.0.0. This module ships with good defaults still, for the common case, but it's easily overridden.
